src/main/java/com/eternalcode/discordapp/DiscordApp.java

```diff
@@ -61,6 +67,12 @@ public static void main(String... args) throws InterruptedException {
         configManager.load(databaseConfig);
         configManager.load(experienceConfig);
 
+        DataManager dataManager = new DataManager(new File("data"));
```

Comment:
Nie rób dataManagera bo robisz to samo co w config manager, lecz pod inna nazwa, bloat code, przerób config manager żeby nie przyjmował new File i twórz foldery wcześniej a później odwołuj się do nich tylko w plikach konfiguracyjnych